hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/acl/prefix/OMPrefixAclRequest.java

```diff
@@ -145,16 +142,21 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
     } finally {
       if (lockAcquired) {
         mergeOmLockDetails(omMetadataManager.getLock()
-            .releaseWriteLock(PREFIX_LOCK, getOzoneObj().getPath()));
+            .releaseWriteLock(PREFIX_LOCK, prefixPath));
       }
       if (omClientResponse != null) {
         omClientResponse.setOmLockDetails(getOmLockDetails());
       }
     }
 
-    OzoneObj obj = getOzoneObj();
+    OzoneObj obj = resolvedPrefixObj;
+    if (obj == null) {
```

Comment:
Is null condition necessary here ? I see a call to `resolvedPrefixObj.getpath ()` on line 81, 82.

Reply:
If `IOException` is thrown during the `getResolvedPrefixObj`, the `resolvedPrefixObj` should be null and we should fall back to the initial ozone object from `getOzoneObj`.